Examples/PythonToInflux/client.py

```diff
@@ -0,0 +1,70 @@
+import argparse
+from influxdb import InfluxDBClient
+import datetime
+
+def main(host='localhost', port=8086):
+    """Instantiate a connection to the InfluxDB."""
+    user = 'root'
+    password = 'root'
+    dbname = 'example'
+    dbuser = 'user'
+    dbuser_password = 'my_secret_password'
+    query = 'select value from cpu_load_short;'
+    json_body = [
+        {
+            "measurement": "cpu_load_short",
+            "tags": {
+                "host": "server01",
+                "region": "us-west"
+            },
+            "time": datetime.datetime.now(),
+            "fields": {
+                "Float_value": 0.64,
+                "Int_value": 3,
+                "String_value": "Text",
+                "Bool_value": True
+            }
+        }
+    ]
+
+    client = InfluxDBClient(host, port, user, password, dbname)
+
+    print("Create database: " + dbname)
+    client.create_database(dbname)
```

Comment:
Maybe it's not that big of a deal because it's local hosted but it's good practice to incorporate error handling when connecting/ instantiating databases (i.e. putting line 33 in a try/ except statement). You can see they also do this in the examples documentation [here](https://github.com/influxdata/influxdb-python/blob/master/examples/tutorial_server_data.py)